lib/middleware/backbone_request_id.js

```diff
@@ -0,0 +1,11 @@
+import Backbone from 'backbone'
+import { wrap } from 'underscore'
+
+module.exports = (req, res, next) => {
+  Backbone.sync = wrap(Backbone.sync, (sync, method, model, options = {}) => {
```

Comment:
I was wondering if the `Backbone.sync` thing would be possible. Oof. Seems even less ideal to have to modify each Backbone call to pass this header in 😫 . Is there any other way? Maybe by hooking into some even lower-level library?

Or, could I override `sync` on each model instead of at a global level?